test/Datadog.Trace.ClrProfiler.IntegrationTests/MongoDbTests.cs

```diff
@@ -35,21 +35,31 @@ public void SubmitsTraces(string packageVersion)
                 Assert.Equal("Samples.MongoDB", firstSpan.Service);
                 Assert.Null(firstSpan.Type);
 
+                int spansWithResourceName = 0;
+
                 for (int i = 1; i < spans.Count; i++)
                 {
                     if (spans[i].Service == "Samples.MongoDB-mongodb")
                     {
                         Assert.Equal("mongodb.query", spans[i].Name);
                         Assert.Equal(SpanTypes.MongoDb, spans[i].Type);
                         Assert.False(spans[i].Tags?.ContainsKey(Tags.Version), "External service span should not have service version tag.");
+
+                        if (spans[i].Resource != null && spans[i].Resource != "mongodb.query")
```

Comment:
nit: Is `spans[i].Resource != "mongodb.query"` necessary?  I thought it would be some valid value or null, but not explicitly `mongodb.query`

Reply:
It's necessary. `mongodb.query` is the operation name, and it's used as default value for the resource name: https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/Span.cs#L320